app/_hub/traceableai/traceableai/overview/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+nav_title: Overview
+title: Overview
+---
+
+Traceable's Kong plugin lets traceable capture a copy of the API traffic that is flowing through Kong's API Gateway. Using this data Traceable is able to create a security posture profile of  APIs hosted on Kong. Traceable plugin is also capable of blocking traffic coming from malicious actors and IPs into Kong API Gateway.
```

Comment:
```suggestion
Traceable's Kong plugin lets Traceable capture a copy of the API traffic that is flowing through {{site.base_gateway}}. 
Using this data, Traceable is able to create a security posture profile of APIs hosted on Kong. 
Based on its findings, the Traceable plugin can also block traffic coming from malicious actors and IPs into {{site.base_gateway}}.
```
Some rephrasing to make the description flow together. Using the `{{site.base_gateway}}` variable, which will resolve to "Kong Gateway" in the output.